lib/suggestion-list-element.coffee

```diff
@@ -162,30 +166,43 @@ class SuggestionListElement extends HTMLElement
     @descriptionContent = @querySelector('.suggestion-description-content')
     @descriptionMoreLink = @querySelector('.suggestion-description-more-link')
 
-  calculateMaxListHeight: ->
-    li = document.createElement('li')
-    li.textContent = 'test'
-    @ol.appendChild(li)
-    itemHeight = li.offsetHeight
-    paddingHeight = parseInt(getComputedStyle(this)['padding-top']) + parseInt(getComputedStyle(this)['padding-bottom']) ? 0
-    @scroller.style['max-height'] = "#{@maxVisibleSuggestions * itemHeight + paddingHeight}px"
-    li.remove()
-
   renderItems: ->
     items = @visibleItems() ? []
     @renderItem(item, index) for item, index in items
     li.remove() while li = @ol.childNodes[items.length]
-    @selectedLi?.scrollIntoView(false)
-
-    @descriptionContainer.style['max-width'] = "#{@offsetWidth}px"
+    @updateDescription()
 
     if @suggestionListFollows is 'Word'
-      firstChild = @ol.childNodes[0]
-      wordContainer = firstChild?.querySelector('.word-container')
-      marginLeft = 0
-      marginLeft = -wordContainer.offsetLeft if wordContainer?
+      wordContainer = @selectedLi?.querySelector('.word-container')
+      marginLeft = -(wordContainer?.offsetLeft ? 0)
```

Comment:
This `offsetLeft` read is now the slowest thing when typing. I really want to move it into the `readUIPropsFromDOM` method, but when I do there is a super annoying jitter. The jitter is from the autocomplete box changing size, waiting an animation frame, reading the `offsetLeft`, waiting another animation frame, then writing the correct margin to the style object. :( :(